src/options.h

```diff
@@ -183,6 +183,8 @@
 /* Subsets cracking mode */
 #define FLG_SUBSETS_CHK			0x0100000000000000ULL
 #define FLG_SUBSETS_SET			(FLG_SUBSETS_CHK | FLG_CRACKING_SET)
+#define FLG_RAIN_CHK			0x2000000000000000ULL //biggest flag yet
+#define FLG_RAIN_SET			(FLG_RAIN_CHK | FLG_CRACKING_SET) 
```

Comment:
Please move these lines to below the `FLG_NO_MASK_BENCH` line, so they come in numerical order